File: fx-api-net/Controllers/ResponseTypes/Quote.cs
namespace FundhouseFx.Controllers.ResponseTypes
{
    public class Quote
    {
        public string BaseCurrency { get; set; }
        
        public string QuoteCurrency { get; set; }
        
        public decimal BaseAmount { get; set; }
        
        public decimal ConvertedAmount { get; set; }
    }
}

File: fx-api-php/packages/FundhouseFx/Api/Responses/Quote.php
<?php
    namespace FundhouseFx\Api\Responses;

    class Quote {
        public string $base_currency;
        public string $quote_currency;
        public float $base_amount;
        public float $converted_amount;

        function __construct(string $base_ccy, string $quote_ccy, float $amount, float $converted_amount)
        {
            $this->base_currency = $base_ccy;
            $this->quote_currency = $quote_ccy;
            $this->base_amount = $amount;
            $this->converted_amount = $converted_amount;
        }
    }
?>